test/Distributed/Runtime/distributed_actor_isolation_passing.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend-emit-module -emit-module-path %t/FakeDistributedActorSystems.swiftmodule -module-name FakeDistributedActorSystems -disable-availability-checking %S/../Inputs/FakeDistributedActorSystems.swift
+// RUN: %target-build-swift -parse-as-library -swift-version 6 -target %target-swift-abi-5.9-triple -I %t %s %S/../Inputs/FakeDistributedActorSystems.swift %S/../Inputs/CustomSerialExecutorAvailability.swift -o %t/a.out
+// RUN: %target-codesign %t/a.out
+// RUN: %target-run %t/a.out
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+// REQUIRES: distributed
+// REQUIRES: concurrency_runtime
+// UNSUPPORTED: back_deployment_runtime
+
+// UNSUPPORTED: back_deploy_concurrency
+// UNSUPPORTED: use_os_stdlib
+// UNSUPPORTED: freestanding
+
+//import StdlibUnittest
+import Distributed
+
+@available(SwiftStdlib 5.9, *)
+distributed
+  actor Tester {
+
+  typealias ActorSystem = LocalTestingDistributedActorSystem
+
+  distributed
+    func check() async {
+    pass()
+    passOptional() // should work, but fails to infer that #isolation is the same as "self"
+
+    pass(isolatedToActor: self.asLocalActor)
+    passOptional(isolatedToActor: self.asLocalActor)
+
+    // Not supported: the parameter must be exactly the 'self.asLocalActor'
+    // as otherwise we don't know where the value came from and if it's really
+    // self we're calling from; We'd need more sophisticated analysis to make it work.
+    //    let myself = self.asLocalActor
+    //    pass(isolatedToActor: myself) // same
```

Comment:
so we can't support this yet... which is fine; the primary usecase is #isolation's default value